src/components/link/link.component.js

```diff
@@ -96,9 +89,7 @@ class InternalLink extends React.Component {
   createLinkBasedOnType = () => {
```

Comment:
I think this would also be a good time to make Links always return an anchor element.